File: GO/Cap04/c04ex02.go
// c04ex02.go

package main
import "fmt"

func main() {
  var A, B, X int

  fmt.Print("Entre o 1o. valor numerico: ")
  fmt.Scan(&A)

  fmt.Print("Entre o 2o. valor numerico: ")
  fmt.Scan(&B)

  if (A > B) {
    X = A
    A = B
    B = X
  }

  fmt.Printf("Os valores sao: %d e %d.\n", A, B)

  fmt.Println()
  fmt.Print("Tecle <Enter> para encerrar... ")
  fmt.Scanln()
}

File: RUBY/Cap04/c04ex02.rb
# c04ex02.rb

#!/usr/bin/env ruby

print "Entre o 1o. valor numerico: "
a = gets.chomp.to_i

print "Entre o 2o. valor numerico: "
b = gets.chomp.to_i

if (a > b)
  x = a
  a = b
  b = x
end

printf "Os valores sao: %d e %d.\n", a, b

puts
print "Tecle <Enter> para encerrar... "
gets
